spring-data-neo4j-rx/src/main/java/org/neo4j/springframework/data/core/mapping/NodeDescriptionStore.java

```diff
@@ -75,4 +77,22 @@ public boolean containsKey(String primaryLabel) {
 		}
 		return null;
 	}
+
+	public NodeDescription<?> deriveConcreteNodeDescription(Neo4jPersistentEntity<?> entityDescription, List<String> labels) {
+		if (labels == null || labels.isEmpty()) {
```

Comment:
So this is what enforces the presence of `@Node`  on the base class, right?

Reply:
This is only a fast return because there is nothing to derive to/from.
The filtering is done in the `Neo4jMappingContext` when the entities get registered as parent/child (or not). A class without a `@Node` annotation will not even get considered in first place.